src/Features/LanguageServer/Protocol/Features/Diagnostics/EngineV2/DiagnosticIncrementalAnalyzer.Executor.cs

```diff
@@ -179,8 +122,8 @@ private static bool CompilationHasOpenFileOnlyAnalyzers(CompilationWithAnalyzers
                 if (compilationWithAnalyzers?.Analyzers.Length > 0)
                 {
                     // calculate regular diagnostic analyzers diagnostics
-                    var resultMap = await _diagnosticAnalyzerRunner.AnalyzeProjectAsync(project, compilationWithAnalyzers,
-                        forcedAnalysis, logPerformanceInfo: false, getTelemetryInfo: true, cancellationToken).ConfigureAwait(false);
```

Comment:
that value eventually flowed into this AnalyzeProjectAsync call.  This is the only place in teh product (not tests) where this method is called.  so the parameter got removed here as well.